src/components/Inputs.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+
+const Inputs = ({ amount, setAmount, setResult }) => {
+
+  const [percentage, setPercentage] = useState(0);
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    setResult((amount * percentage) / 100)
+  }
+
+  return (
+    <form onSubmit={handleSubmit} className="app__form">
+      <input type="number" name="amount" onChange={e => setAmount(e.target.value)} required className="app__form--input" placeholder="Amount $" />
```

Comment:
It looks like your total bill doesn't account for cents. I believe that it would have been best to use a text input versus a number input to account for this. You could use error handling to ensure you dont recieve any text in the input, such as isNaN()